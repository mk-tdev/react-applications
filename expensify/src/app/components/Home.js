import React from "react";
import { NavLink } from "react-router-dom";
import ConnectedExpensesList from "./ExpensesList";
import ExpensesListFilter from "./ExpensesListFilter";

const Home = () => {
  return (
    <div>
      <h2>Expenses List</h2>
      <ExpensesListFilter />
      <ConnectedExpensesList />

      <NavLink to="/edit/2" activeClassName="is-active" exact>
        Help
      </NavLink>
    </div>
  );
};

export default Home;
