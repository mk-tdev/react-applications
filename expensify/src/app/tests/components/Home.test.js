import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";

configure({ adapter: new Adapter() });

import Home from "../../components/Home";

test("should render Home", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});
