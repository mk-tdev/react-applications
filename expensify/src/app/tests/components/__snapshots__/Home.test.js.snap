// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home 1`] = `
<div>
  <h2>
    Expenses List
  </h2>
  <Connect(ExpensesListFilter) />
  <Connect(ExpensesList) />
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/edit/2"
  >
    Help
  </NavLink>
</div>
`;
